Documentation/RC1SupportedScenarios.md

```diff
@@ -0,0 +1,24 @@
+WCF in RC1
+===========================
+
+WCF Features
+------------
+
+
+![Image](http://findicons.com/files/icons/573/must_have/16/check.png) -- Supported: Release quality
+
+![Image](http://findicons.com/files/icons/1155/diagona/16/104.png) --  Supported: Not fully tested
+
+![Image](http://findicons.com/files/icons/573/must_have/16/add.png) -- Not Yet Supported: Future release
+
+![Image](http://findicons.com/files/icons/573/must_have/16/delete.png) -- Not Supported: Will not support
+
+
+
+|Scenario |UWP   |Windows |Linux |Mac OS X  |
+|---------|------|--------|------|----------|
```

Comment:
@roncain you have a good point about Linux distro. I think we should explicitly list the one we tested. If we can cover more Linux distro's in the future, we can add more columns to the table. If it becomes too many columns in the future, we will need to think about a better way, but for now I hope we can keep it simple.